README.md

```diff
@@ -1,24 +1,67 @@
-# README
+# Hello-rails-back-end
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+Set up a "Hello world" project with Ruby on Rails as back-end
 
-Things you may want to cover:
+You will create a Ruby on Rails application that allows the user to:
 
-* Ruby version
+This app was created with Rails
 
-* System dependencies
+# Link to React app
 
-* Configuration
+https://github.com/fibini/hello-react-front-end
```

Comment:
- [ ] OPTIONAL
The correct link to the frontend should be 
_https://github.com/fibini/hello-react-front-end/tree/feature_

Kindly update the same or merge to update the current link provided